src/main/java/fashionable/simba/yanawaserver/matching/domain/MatchingReview.java

```diff
@@ -19,10 +27,16 @@ public String getDetails() {
         return details;
     }
 
-    public MatchingReview(Long writerId, Long partnerId, String details) {
-        if(writerId == null || partnerId == null) {
-            throw new NoPlayerDataException("사용자 정보가 없어 리뷰를 작성할 수 없습니다.");
-        }
+    public MatchingReview(Long matchingId, Long writerId, Long partnerId, String details) {
+        this.matchingId = matchingId;
+        this.writerId = writerId;
+        this.partnerId = partnerId;
+        this.details = details;
+    }
```

Comment:
```suggestion
    public MatchingReview(Long matchingId, Long writerId, Long partnerId, String details) {
        this(null, matchingId, writerId, partnerId, details)
    }
```

이렇게 관리하는게 가독성도 좋고, 생성자들을 관리하기도 쉽습니다. ☺️